pillar/fluentd/mitx.sls

```diff
@@ -68,6 +69,7 @@ fluentd:
             - format1: '/^(?<time>\w{3}\s+\d{1,2}\s+\d{1,2}:\d{2}:\d{2}) (?<hostname>[^ ]+?) \[service_variant=(?<service_variant>\w+?)\]\[(?<namespace>[a-zA-Z._-]+?)\]\[env:(?<logging_env>[a-zA-Z-_.]+)\]\ (?<log_level>\w+) \[[^ ]+\s+\d+\] \[(?<filename>[a-zA-Z0-9-_.]+):(?<line_number>\d+)\] - (?<message>.*)$/'
             - time_format: '%b %d %H:%M:%S'
             - multiline_flush_interval: '5s'
+        {% if 'mitxpro' not in ENVIRONMENT %}
```

Comment:
Rather than being conditional on the environment this might be more robust if we make it conditional on the presence of the directory, e.g. `if salt.file.directory_exists('/edx/var/log/gr`)`